<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HealthFacilityEditDialog</class>
 <widget class="QDialog" name="HealthFacilityEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактирование данных о мед. учреждении</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Название</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="m_name"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Сокращенное название</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="m_shortName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="toolTip">
        <string>Идентификационный номер налогоплательщика</string>
       </property>
       <property name="text">
        <string>ИНН</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="toolTip">
        <string>Код причины постановки на учёт</string>
       </property>
       <property name="text">
        <string>КПП</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="toolTip">
        <string>Общесоюзный классификатор отраслей народного хозяйства</string>
       </property>
       <property name="text">
        <string>ОКОНХ</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="toolTip">
        <string>Общероссийский классификатор объектов административно-территориального деления</string>
       </property>
       <property name="text">
        <string>ОКАТО</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="toolTip">
        <string>Общероссийский классификатор предприятий и организаций</string>
       </property>
       <property name="text">
        <string>ОКПО</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_8">
       <property name="toolTip">
        <string>Основной государственный регистрационный номер</string>
       </property>
       <property name="text">
        <string>ОГРН</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="m_inn"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="m_kpp"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="m_okonh"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="m_okato"/>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="m_okpo"/>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="m_ogrn"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_ok">
       <property name="text">
        <string>ОК</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_cancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
